docs/sc-tools/codegen_removal.md

```diff
@@ -0,0 +1,12 @@
+# codegen-removal
+
+This is a program that translates sc-machine usage from old format into new format.
+
+to run you can use script
+```shell
+./scripts/codegen-removal/remove_codegen.sh /absolute/path/to/folder/with/project
```

Comment:
It can be relative path?

Reply:
no it cannot be a relative path

Reply:
why?

Reply:
script may be invoked from anywhere and relative path for that place may differ from relative path for script location and it is not necessary to support that

Reply:
I think support for the relative path should be made. We support it everywhere

Reply:
![image](https://github.com/user-attachments/assets/91dd5dac-30ab-42bf-8147-3a9ab8cad842)
not everywhere

Reply:
turns out it did work with not absolute paths